docs/sources/setup/install/helm/install-microservices/_index.md

```diff
@@ -165,7 +170,7 @@ It is not recommended to run scalable mode with `filesystem` storage. For the pu
 
 ## Object Storage Configuration
 
-After testing Loki with MinIO, it is recommended to configure Loki with an object storage provider. The following examples shows how to configure Loki with different object storage providers:
+After testing Loki with MinIO, we recommend to configure Loki with an object storage provider. The following examples shows how to configure Loki with different object storage providers:
```

Comment:
suggest link on word MinIO - https://min.io/docs/minio/kubernetes/upstream/index.html

very minor point but it's more than possible people won't know what this is.